CMakeLists.txt

```diff
@@ -11,7 +11,6 @@ set(QUIC_BUILD_TEST OFF CACHE BOOL "Don't need the tests")
 set(QUIC_BUILD_PERF OFF CACHE BOOL "Don't need perf")
```

Comment:
TOOLS, TEST and PERF are al disables by default in the latest msquic so you can remove them from here